### PR TITLE
completed html-syntax exercise, avallon smith

### PR DESCRIPTION
**Pull request by avallonsmith**
image

### CODE REVIEW

File: html-syntax/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML-Syntax</title>
+</head>
+<body>
+  <img src="https://i.imgur.com/wFZcU0Q.png">
```
Comment (Zhiwutian) [inline html-syntax/index.html:10]: don't forget your alt text!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
tiny update!

File: html-syntax/index.html (hunk 1 above)
Comment (avallonsmith) [reply html-syntax/index.html:10]: whoops! will fix that right now, thank you!

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**